### PR TITLE
2.0 Audit Logs

### PR DESCRIPTION
**Pull request by go0ty**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -136,6 +136,11 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.venice.piazza</groupId>
+			<artifactId>pz-jobcommon</artifactId>
+			<version>2-0-0-7-gb51fe13</version>
```
Comment (fsufitch) [inline pom.xml]: Is this version pegging going to be problematic for maintainability?

File: src/main/java/org/venice/beachfront/bfapi/services/JobService.java [hunk 2]
```diff
@@ -106,12 +115,12 @@ public Job createJob(String jobName, String creatorUserId, String sceneId, Strin
 		Scene scene = null;
 		try {
 			scene = sceneService.getScene(sceneId, planetApiKey, true);
-		} catch (Exception exception) {
+		} catch (UserException exception) {
```
Comment (fsufitch) [inline src/main/java/org/venice/beachfront/bfapi/services/JobService.java]: Why are we erasing and re-emitting UserExceptions? The method already `throws UserException` so why not just... not catch these?

File: pom.xml (hunk 1 above)
Comment (go0ty) [reply pom.xml]: This is how all of our libraries are versioned (apps, too, for that matter). Updates contain potentially breaking changes, so we do not want to automate with some LATEST version -- if that's what you mean.

File: src/main/java/org/venice/beachfront/bfapi/services/JobService.java (hunk 2 above)
Comment (go0ty) [reply src/main/java/org/venice/beachfront/bfapi/services/JobService.java]: Good point. This is a relic of before the Scene Service switched over to user exceptions.  I will remove the unnecessary try/catch block.

**Pull request merged**